be/src/storage/rowset/column_reader.h

```diff
@@ -87,14 +87,16 @@ struct NgramBloomFilterReaderOptions;
 // This will cache data shared by all reader
 class ColumnReader {
     struct private_type;
+    struct SubReaderId;
 
 public:
     // Create and initialize a ColumnReader.
     // This method will not take the ownership of |meta|.
     // Note that |meta| is mutable, this method may change its internal state.
     //
     // To developers: keep this method lightweight, should not incur any I/O.
-    static StatusOr<std::unique_ptr<ColumnReader>> create(ColumnMetaPB* meta, Segment* segment);
+    static StatusOr<std::unique_ptr<ColumnReader>> create(ColumnMetaPB* meta, Segment* segment,
```

Comment:
And you need to also comment that not all client need to parse column argument.

Reply:
> What the necessarily of the new argument `column`? Why can't you generate ColumnReader from meta itself? You need to explain it in the above comments.

Because we need to obtain the name of the sub_column, but the name is not stored in the metadata. And I will add more comment for the new added argument.